tutorials/getting_started/rules_advanced.md

```diff
@@ -231,6 +388,52 @@ That keeps it from being modified or overwritten by `npm` when upgrading or inst
 There are several third party openHAB specific libraries on `npm` already.
 Search the forum for details.
 
+:::
+
+::: tab JRuby
+
+You can install JRuby-compatible Gems from [rubygems.org](https://rubygems.org) by listing them in the JRuby Script add-on [configurations](https://openhab.github.io/openhab-jruby/main/index.html#configuration).
+Alternatively, [inline Bundler](https://bundler.io/guides/bundler_in_a_single_file_ruby_script.html) is available from inside your script.
+
+```ruby
+gemfile do
+  source "https://rubygems.org"
+  gem "faraday", "~> 2.12", ">= 2.12.2"
+end
+```
+
+[Personal libraries](https://openhab.github.io/openhab-jruby/main/index.html#shared-code) that contain your custom reusable constants, functions, data structures, and classes can be created and used inside your Script Actions and Conditions.
+Simply save a `.rb` file that contain your code in `$OH_CONF/automation/ruby/lib/` folder, then `require` it in your script.
```

Comment:
```suggestion
Simply save a `.rb` file that contains your code in `$OH_CONF/automation/ruby/lib/` folder, then `require` it in your script.
```

Also add a suggestion that you can add your custom library to the auto-requires in the add-on settings.